wemake_python_styleguide/visitors/ast/naming.py

```diff
@@ -45,7 +45,7 @@
 class _NameValidator(object):
     """Utility class to separate logic from the naming visitor."""
 
-    _variable_names_blacklist: Optional[Set[str]]
+    variable_names_blacklist: Set[str]
```

Comment:
Why is it public?

Reply:
To be honest I don't know :D I found it more proper/clear when I was writing this code. I will correct it in following commit